Assets/MixedRealityToolkit.Providers/WindowsMixedReality/WindowsMixedRealityEyeGazeDataProvider.cs

```diff
@@ -69,7 +69,7 @@ public bool CheckCapability(MixedRealityCapability capability)
             if (WindowsApiChecker.UniversalApiContractV8_IsAvailable)
             {
 #if WINDOWS_UWP
-                return ((capability == MixedRealityCapability.EyeTracking) && EyesPose.IsSupported());
+                return capability == MixedRealityCapability.EyeTracking && EyesPose.IsSupported();
```

Comment:
```suggestion
                return (capability == MixedRealityCapability.EyeTracking) && EyesPose.IsSupported();
```
I actually liked that one parenthesis - improves readability imo